apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/index.tsx

```diff
@@ -65,9 +68,10 @@ const SharingModalInner: React.FC = () => {
 	const shouldShowVideoCelebrationModal =
 		useShouldShowVideoCelebrationModal( isCurrentPostPublished );
 
-	const [ isOpen, setIsOpen ] = useState( false );
+	const [ isOpen, setIsOpen ] = useState( true );
```

Comment:
I imagine this might be for convenience during development?

In testing this the modal was present as soon as I opened the editor.

Reply:
Yep, this is only here for testing so we can open the post published dialog easily